rfc/5/versions/1/index.md

```diff
@@ -0,0 +1,977 @@
+
+````{admonition} Example
+
+Coordinate Systems metadata example
+
+```json
+{
+    "name" : "volume_micrometers",
+    "axes" : [
+        {"name": "z", "type": "space", "unit": "micrometer"},
+        {"name": "y", "type": "space", "unit": "micrometer"},
+        {"name": "x", "type": "space", "unit": "micrometer"}
+    ]
+}
+```
+````
+
+The axes of a coordinate system (see below) give information
+about the types, units, and other properties of the coordinate system's dimensions.
+Axis `name`s may contain semantically meaningful information, but can be arbitrary.
+As a result, two coordinate systems that have identical axes in the same order 
+ay not be "the same" in the sense that measurements at the same point
+refer to different physical entities and therefore should not be analyzed jointly.
+Tasks that require images, annotations, regions of interest, etc.,
+SHOULD ensure that they are in the same coordinate system (same name, with identical axes)
+or can be transformed to the same coordinate system before doing analysis.
+See the example below.
+
+#### "axes" metadata
+
+"axes" describes the dimensions of a coordinate systems
+and adds an interpretation to the samples along that dimension.
+
+It is a list of dictionaries,
+where each dictionary describes a dimension (axis) and:
+- MUST contain the field "name" that gives the name for this dimension.
+  The values MUST be unique across all "name" fields.
+- SHOULD contain the field "type".
+  It SHOULD be one of the strings "array", "space", "time", "channel", "coordinate", or "displacement"
+  but MAY take other string values for custom axis types that are not part of this specification yet.
+- MAY contain the field "discrete".
+  The value MUST be a boolean,
+  and is `true` if the axis represents a discrete dimension.
+- SHOULD contain the field "unit" to specify the physical unit of this dimension.
+  The value SHOULD be one of the following strings,
+  which are valid units according to UDUNITS-2.
+    - Units for "space" axes: 'angstrom', 'attometer', 'centimeter', 'decimeter', 'exameter', 'femtometer', 'foot', 'gigameter', 'hectometer', 'inch', 'kilometer', 'megameter', 'meter', 'micrometer', 'mile', 'millimeter', 'nanometer', 'parsec', 'petameter', 'picometer', 'terameter', 'yard', 'yoctometer', 'yottameter', 'zeptometer', 'zettameter'
+    - Units for "time" axes: 'attosecond', 'centisecond', 'day', 'decisecond', 'exasecond', 'femtosecond', 'gigasecond', 'hectosecond', 'hour', 'kilosecond', 'megasecond', 'microsecond', 'millisecond', 'minute', 'nanosecond', 'petasecond', 'picosecond', 'second', 'terasecond', 'yoctosecond', 'yottasecond', 'zeptosecond', 'zettasecond'
+- MAY contain the field "longName".
+  The value MUST be a string,
+  and can provide a longer name or description of an axis and its properties.
+
+If part of metadata, the length of "axes" MUST be equal to the number of dimensions of the arrays that contain the image data.
```

Comment:
Part of which metadata? Because this isn't a constraint on a a single Axes object, I think it makes sense to move the constraint to the structure where it applies.

Reply:
👍